.gitignore

```diff
@@ -1 +1,3 @@
 .DS_Store
+server/omsupply-database-shm
```

Comment:
i had this created in other places, not just in server

Reply:
Yes, it's created on all the test db's too. They have .sqlite-shm so I can add a more generic ignore...

Reply:
Yes please